src/main/kotlin/org/partiql/scribe/ExpandStruct.kt

```diff
@@ -1,53 +0,0 @@
-package org.partiql.scribe
```

Comment:
(self-review) Function deleted since each target will have it's own logic to expand structs for tupleunion.